include/aikido/rviz/WorldInteractiveMarkerViewer.hpp

```diff
@@ -38,8 +38,8 @@ class WorldInteractiveMarkerViewer : public InteractiveMarkerViewer
   /// Thread target for auto-updating the viewer
   void autoUpdate();
 
-  /// Mapping of Skeleton names to SkeletonMarkers
-  std::map<std::string, SkeletonMarkerPtr> mSkeletonMarkers;
+  /// Mapping of Skeletons to SkeletonMarkers
+  std::map<dart::dynamics::SkeletonPtr, SkeletonMarkerPtr> mSkeletonMarkers;
```

Comment:
Although this is a stopgap solution (over [using callbacks](https://github.com/personalrobotics/aikido/pull/252#issue-269541118)), we might want to use `std::weak_pointer<...>` here because this class doesn't need to have the ownership as it just uses the skeletons only when they still exist in `mWorld`.